AnnService/inc/Core/Common/PQQuantizer.h

```diff
@@ -377,7 +389,22 @@ namespace SPTAG
         inline SizeType PQQuantizer<T>::m_DistIndexCalc(SizeType i, SizeType j, SizeType k) const {
             return m_BlockSize * i + j * m_KsPerSubvector + k;
         }
+
+	template <typename T>
+        float* PQQuantizer<T>::GetCosineDistanceTables() {
+		printf("Getting pointer to cosine distance tables...\n");
+		// SEG FAULTS HERE, HOW CAN WE ACCESS THIS?
```

Comment:
Does it still segfault, or we can remove this

Reply:
Was solved, removed comment and print statement.